dbms/tests/integration/test_row_policy/single_table_name.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
```

Comment:
It doesn't seem to be easy to understand the names of these test xml files: `custom` doesn't really mean "custom" because all names are "custom" in some way, and `single` doesn't really mean "single", because there are three tables there.

I think it's better to rename these files to make the tests clearer, e.g.
`single_table_name.xml` -> `tag_with_table_name.xml`,
`custom_table_name.xml` -> `multiple_tags_with_table_names.xml`.